api/src/main/java/org/jfrog/artifactory/client/model/RepositorySummary.java

```diff
@@ -6,22 +6,79 @@
  * @author Aviad Shikloshi
  */
 @JsonIgnoreProperties(ignoreUnknown = true)
-public interface RepositorySummary {
+public class RepositorySummary {
```

Comment:
What is the reason for changing this interface to class?

Reply:
The serialisation / deserialisation don't work anymore with new version of third party library. The code in RepositorySummaryDeserialize with the TypeReference on a list doesn't compile anymore.
It's sure that jackson need to know a concrete type. I choose the same approach that have CopyMoveResultMessage and CopyMoveResultReport. The CopyMoveResultMessage  is not an interface.
Was my explanation good enough?
I understand the basic idea of interface for api but not for dto object.

Reply:
Yes. Thank you